docs/sphinx/source/installation.rst

```diff
@@ -38,11 +56,17 @@ For best compatibility, deploy in an `Anaconda 2.7` environment, or run::
 
 STEP 2
 ~~~~~~
-Move gencumulativesky.exe
+Windows:
 
 * Copy gencumulativesky.exe from the repo's ``/bifacial_radiance/data/`` directory and copy into your Radiance install directory.
   This is typically found in ``/program files/radiance/bin/``.  
  
+Linux/Mac OSX:
+
+* A make_gencumskyexe.py script that builds gencumsky executable on pultiple platforms (Linux, Mac OSX, and Windows too) is included in bifacial_radiance release. More details on the use of this script on readme.txt or on thread (:issue:`1821`).
```

Comment:
Actually, we inlcude a linux executable "gencumulativesky" which should be moved to the Radiance PATH just like with the gencumulative.exe.  The make_gencumskyexe.py generates both the Windows and Linux executables, but is NOT a pre-requisite for installation.

Reply:
Seems you updated this text properly, closing this conversation.